drivers/storage/portworx/component/telemetry.go

```diff
@@ -90,8 +96,11 @@ const (
 	stagingArcusRestProxyURL        = "rest.staging-cloud-support.purestorage.com"
 	stagingArcusRegisterProxyURL    = "register.staging-cloud-support.purestorage.com"
 
-	// port for log uploader, avoid 1970 as used by old ccm service, to be configurable in the future
-	logUploaderPort = 9090
+	// Port offset for telemetry components
+	ccmListeningPortOffset          = 89
+	collectorCloudSupportPortOffset = 999
```

Comment:
can we just use port number, the offset is harder to track and avoid conflict.

Reply:
port number variables are defined locally currently, we cannot define const for those port values as we need stc to get the start port first, and it could be updated. what I could do is define some default ports, check whether the start port is the default one, if not we use another function to get the shifted ports. wdyt?

Reply:
> can we just use port number, the offset is harder to track and avoid conflict.

eventually I think we should have one place to track all port offsets to avoid conflict.

Reply:
any reason to choose 89,999,3000, instead of 1,2,3,4,5?

Reply:
@haibinxie for ccm listening port we need to be consistent with portworx side, it used to be 9090 by default, and will shift with the start port 9001, which we pass with a `-r` arg. we need changes on both operator and px side.
collector we hard-coded 10000 as default previously, so still use 10000 by default, to avoid port conflict we need to shift it as well
for register and phonehome, we just chose a higher value as default in the template to avoid conflict, since when using custom proxy, by default all 12001~12006 will be used.